src/components/Filter.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+
+export default class Filter extends React.Component {
+
+state = {
+    products: [],
+    inputValueMin: '',
+    inputValueMax: '',
+    inputCategory: ''
+}
+
+componentDidMount() {
+    this.props.products
+    this.setState({products: this.state.products})
+}
+
+filter() {
+    showOrderedFilters = () => {
+        if(this.props.minFilter > 0 || this.props.maxFilter > 0 ) {
```

Comment:
esse this.props.minFilter e this.props.maxFilter não existem, o SearchAd apenas passou para cá this.props.products e this.props.filter